uptane-standard.md

```diff
@@ -238,23 +248,23 @@ A standards document that can guide the safe design, integration and deployment
 * This added design flexibility, however, could be a liability if the framework is implemented incorrectly.
 * Standardization of crucial steps in the design, implementation and use of Uptane can assure that customizability does not impact security or functionality.
 
-## Scope of Standards Coverage
+## Scope of Standards coverage
```

Comment:
Should 'standards' also be lowercase here?

Reply:
I think nit, because its referring to this particular standard document.

Reply:
Standards here is a proper noun, which I do capitalize throughout. Would it look less weird if I changed it to Uptane Standards coverage?